### PR TITLE
Create Events and math operations

### PR DESCRIPTION
**Pull request by andersonlebon**
- [ ] Copy the files `calculator.js` and `operate.js` into a logic/ directory in your project.
- [ ] Analyze the files calculator.js and operate.js, and understand what they do.
- [ ] Import the files in my Calculator component.
- [ ] Implement the event handlers using the math logic from calculator.js and operate.js in my React component.
- [ ] Test my app in the browser. It should be able to perform math operations.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,50 +1,178 @@
...
+            +/-
+          </li>
+          <li
+            className="cal-item"
+            id="%"
+            onClick={this.handleCalculate}
+            aria-hidden="true"
+          >
+            %
+          </li>
```
Comment (MahmoudBakr23) [inline src/components/Calculator.js:53]: - [ ] Please double-check on this button's functionality because it doesn't seem to output the right results. 🙏 😄

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @andersonlebon 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct
- Descriptive PR headline, summary, and commit messages

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (andersonlebon) [reply src/components/Calculator.js:53]: Hello, @MahmoudBakr23 I think that is not my mistake that is the way the function `calculate` that we copied should function. and I think I am not allowed to change the content of that function.

Comment (andersonlebon) [reply src/components/Calculator.js:53]: KIndly everything is working well except the situation when we have an existing operation and you click on a new operation.

**Comment by Davidosky007** [comment]
## Hi @andersonlebon 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
3# Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**